persistent-mysql/test/main.hs

```diff
@@ -181,6 +183,9 @@ main = do
 
     MigrationIdempotencyTest.specsWith db
     CustomConstraintTest.specs db
+    -- MySQL currently fails this test, because it has a limit of 64 characters for identifiers
+    -- TODO: implement automatic truncation for too long foreign keys
+    -- LongIdentifierTest.specsWith db
```

Comment:
Instead of commenting this out, please use a [`pendingWith`](https://www.stackage.org/haddock/lts-14.19/hspec-2.7.1/Test-Hspec.html#v:pendingWith) or [`xdescribe`](https://www.stackage.org/haddock/lts-14.19/hspec-2.7.1/Test-Hspec.html#v:xdescribe) call to make this more clear at runtime :)

```suggestion
    xdescribe "MySQL currently fails because of 64 character limit for identifiers (# issue number)" $
        LongIdentifierTest.specsWith db
```